.azure-pipelines/util/test-steps.yml

```diff
@@ -6,7 +6,6 @@ parameters:
   powerShellPlatform: ''
 
 steps:
-- template: get-github-pat-steps.yml
```

Comment:
Last question: this step was deleted from test and analyze. Was it because those jobs can share the token from the **build** job?

Reply:
No. It's because the test and analysis steps do not use GitHub PAT at all. We pass the `OCTOKITPAT` value to them but they do not need it.